src/main/java/org/sagebionetworks/web/client/PortalGinModule.java

```diff
@@ -163,342 +46,48 @@
 import org.sagebionetworks.web.client.widget.biodalliance13.editor.BiodallianceSourceEditorViewImpl;
 import org.sagebionetworks.web.client.widget.breadcrumb.BreadcrumbView;
 import org.sagebionetworks.web.client.widget.breadcrumb.BreadcrumbViewImpl;
-import org.sagebionetworks.web.client.widget.clienthelp.ContainerClientsHelp;
-import org.sagebionetworks.web.client.widget.clienthelp.ContainerClientsHelpImpl;
-import org.sagebionetworks.web.client.widget.clienthelp.FileClientsHelp;
-import org.sagebionetworks.web.client.widget.clienthelp.FileClientsHelpImpl;
-import org.sagebionetworks.web.client.widget.clienthelp.FileViewClientsHelp;
-import org.sagebionetworks.web.client.widget.clienthelp.FileViewClientsHelpImpl;
-import org.sagebionetworks.web.client.widget.discussion.DiscussionThreadListItemWidgetView;
-import org.sagebionetworks.web.client.widget.discussion.DiscussionThreadListItemWidgetViewImpl;
-import org.sagebionetworks.web.client.widget.discussion.DiscussionThreadListWidgetView;
-import org.sagebionetworks.web.client.widget.discussion.DiscussionThreadListWidgetViewImpl;
-import org.sagebionetworks.web.client.widget.discussion.ForumWidgetView;
-import org.sagebionetworks.web.client.widget.discussion.ForumWidgetViewImpl;
-import org.sagebionetworks.web.client.widget.discussion.NewReplyWidgetView;
-import org.sagebionetworks.web.client.widget.discussion.NewReplyWidgetViewImpl;
-import org.sagebionetworks.web.client.widget.discussion.ReplyWidgetView;
-import org.sagebionetworks.web.client.widget.discussion.ReplyWidgetViewImpl;
-import org.sagebionetworks.web.client.widget.discussion.SingleDiscussionThreadWidgetView;
-import org.sagebionetworks.web.client.widget.discussion.SingleDiscussionThreadWidgetViewImpl;
-import org.sagebionetworks.web.client.widget.discussion.SubscribersWidgetView;
-import org.sagebionetworks.web.client.widget.discussion.SubscribersWidgetViewImpl;

```

Comment:
Again, nice cleanup, but as soon as anyone runs "Organize Imports" within the IDE all of these will show up again.

Reply:
I fixed all my imports to match that of Eclipse's auto importing behavior.